src/components/fields/schemaFields/widgets/varPopup/menuFilters.ts

```diff
@@ -241,12 +244,7 @@ export function defaultMenuOption(
     return null;
   }
 
-  if (
-    likelyVariable == null ||
-    likelyVariable === "@" ||
-    !likelyVariable.startsWith("@") ||
-    toPath(likelyVariable).length === 0
-  ) {
+  if (isTextOrNull(likelyVariable) || toVarPath(likelyVariable).length === 0) {
```

Comment:
nit: reuse the parts variable?
```suggestion
  if (isTextOrNull(likelyVariable) || parts.length === 0) {
```